app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,29 @@
+class ReviewsController < ApplicationController
+  def create
+    @drink = Drink.find(params[:drink_id])
+    @review = @drink.reviews.build(review_params)
+    if current_user
+      @review.user_id = current_user.id
+      if @review.save
+        flash[:notice] = "Successfully added your review"
+        redirect_to drink_reviews_path(@drink)
+      else
+        render 'drinks/show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.